src/rebuild-parse-cache/index.ts

```diff
@@ -29,9 +29,7 @@ async function main(): Promise<void> {
   const WORKSPACE = process.env.WORKSPACE;
 
   const [dockerfileEnhanced, previousDockerfileEnhanced] = await Promise.all([
```

Comment:
(Nit) We can probably rename these too to avoid any confusion about what "enhanced" means